doc/manual/manual.tex

```diff
@@ -7101,6 +7101,73 @@ \subsubsection{Simple convection in a spherical 3d shell}
+\label{sec:cookbooks-shell_3d_postprocess.prm}
+\textit{This section was contributed by Jacqueline Austermann, Ian Rose and Shangxin Liu}
+
+
+There are several postprocessors that can be used to turn the the velocity 
+and pressure solution into quantities that can be compared to surface 
+observations. In this cookbook we introduce two postprocessors: dynamic 
+topography and the geoid. We initialize the model with a harmonic 
+perturbation of degree 4 and order 2 and calculate the
+instantaneous solution. Analogous to the previous setup we use a spherical
+shell geometry model and a simple material model. 
+
+The relevant section in the input file that determines the postprocessed
+output is as follows:
+
+\lstinputlisting[language=prmfile]{cookbooks/shell_3d_postprocess/shell_3d_postprocess.part.prm.out}
+
+This initial condition results in distinct flow cells that cause local up- and
+downwellings (Figure~\ref{fig:pp}). This flow deflects the top and bottom boundaries
+of the mantle away from their reference height, a process known as dynamic topography.
+The deflection of the surfaces and density perturbations within the mantle also 
+cause a perturbation in the gravitational field of the planet relative to the
+hydrostatic equilibrium ellipsoid. 
+
+\paragraph{Dynamic topography at the surface and core mantle boundary.}
+Dynamic topography is calculated at the surface and bottom of the domain 
+through a stress balancing approach where we assume that the radial stress at 
+the surface is balanced by excess (or deficit) topography. We use the 
+consistent boundary flux (CBF) method to calculate the radial stress 
+at the surface \cite{ZGH93}. For the 
+bottom surface we define positive values as up (out)
+and negative values are in (down), analogous to the deformation of the upper 
+surface. Dynamic topography can be outputted in text format (which writes the 
+Euclidean coordinates followed by the corresponding topography value) or as 
+part of the visualization. The upwelling and downwelling flow along the 
+equator causes alternating topography high and lows at the top and
+bottom surface (Figure~\ref{fig:pp}). 
+In Figure~\ref{fig:pp} c, d we have subtracted the mean dynamic topography from
+the output field. Since mass is conserved within the Earth, the mean dynamic topography
+should always be zero, however, the outputted values might not fullfill this 
+constraint if the resolution of the model is not high enough to provide an
+accurate solution. This cookbook only uses a refinement of 2, which is relatively
+low resolution. 
+
+\paragraph{Geoid anomalies.}
+Geoid anomalies are perturbations of the gravitational equipotential surface that 
+are due to density variations within the mantle as well as deflections of the 
+surface and core mantle boundary. The geoid postprocessor subtracts the mean
+dynamic topography at the top and bottom surface when evaluating its contribution
+to the gravity field. 
```

Comment:
Good catch, yes and no. I previously looked at the geoid postprocessor and it says in line 271 "Subtract the average dynamic topography" so I assumed that's what is done. Looking more carefully at it now it seems that this isn't actually done (probably a remnant from an earlier version). But, the default minimum degree is 2 so that removes the mean. I think that's what's going on, does that sound right? If so I can clarify this in the text.